src/Components/ContextMenu.re

```diff
@@ -36,8 +36,7 @@ type item('data) = {
 type placement = {
   x: int,
   y: int,
-  originX: [ | `Left | `Middle | `Right],
-  originY: [ | `Top | `Middle | `Bottom],
+  orientation: ([ | `Top | `Middle | `Bottom], [ | `Left | `Middle | `Right]),
```

Comment:
This is much better terminology I think.

Reply:
Looks pretty clear to me!